README.md

```diff
@@ -8,7 +8,7 @@ Implement the ability to reorder the goods with the given buttons.
     - remove `is-light` class from the button when this order is applied;
 1. `Sort by length` button should sort the goods by length.
     - remove `is-light` class from the button when this order is applied;
-1. `Reverse` button changes the current order to the opposite one. 
+1. `Reverse` button changes the current order to the opposite one.
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use `git add src/.` & `git commit -m 'your message'`